Automattic/gravatar-enhanced
# WooCommerce: WP-Admin: Avatar on Customers page

## Description

In this PR, we add avatars to the Woo Customers page:

![image](https://github.com/user-attachments/assets/ad337a5d-b5a5-4d06-9711-dcaba4b66140)

By hovering the avatars, we show the Gravatar user hovercard if it is available.

## Testing instructions

* Install a local WP site (Using https://localwp.com/ is an easy way to do that)
* Once you have a local site running, go to the wp-admin and install the WooCommerce plugin
* Navigate to `/wp-admin/user-new.php` and add one or two users with the `Customer` role
  * Use a valid Gravatar email for one of them so you can see the integration working
*  Navigate to `/wp-admin/admin.php?page=wc-admin&path=%2Fcustomers` and check if the Gravatar avatars appear as expected
* Play with the customer table (sorting, filters, etc) and check if everything works as expected

Props @jcheringer for working on the JS side.
